### ISSUE TITLE
[Question] How to get Primordial Pearls ? 

### ISSUE THREAD

**Issue by Mr3DAlien** [Azanor/thaumcraft-beta#1527]
Hello so the title is the Question.

I made a causality collapser and destroyed 1000 rifts, but non of them dropped a pearl.

**Comment by lOmicronl** [comment]
1000 rifts? I highly doubt it =P

In any case, as far as I understand it, it works like this:

When a rift collapse occurs (1 in 6 chance when closing naturally, 100% chance when using causality collapser), there is an x% chance for a primordial pearl to drop, where x is the size of the rift from 1 to 10. So if you just spam-create-and-close small rifts, your chances are literally 1% per event. If you invest the flux and time to grow a rift to maximum size, your chance is 10%, but it takes a long time for each one.

Of course, I have never managed to get a primordial pearl to drop either, from the eight large rifts I've tried (all sizes 8-9).

I've stopped trying since, because I can't really be bothwered to invest an entire evening's playtime into 2-3 dice rolls of disappointment, during which I can't do anything else because I have to hang around to keep the rifts loaded and feed the crucibles, but cannot be near my base because I don't want flux there. Instead, I will simply wait until this placeholder mechanic is changed to something more reliable and fun, like for example killing a boss as in previous versions.

**Comment by Mr3DAlien** [comment]
This are big rifts

https://www.youtube.com/upload

**Comment by lOmicronl** [comment]
Yeah, that looks like simple bad luck. Nothing you can do about it - the chance is just that low.

Though it's possible that the second rift dropped one, and it was destroyed by the third rift immediately spawning in the same space. Some, if not all of the void seeds definitely got destroyed, judging by the amount in your inventory afterwards. Just 20 seeds from 3 large rifts, while every rift you collapse after that yields 11.

Or, it could actually be bugged. Problem is, with the drop chance this low, a proof is borderline impossible just from repeating negative results. So we can't say for sure, unless we get really, really large sample sizes from multiple people.

If you don't want to keep repeating this exercise, you could spawn one in. Pay for it by legitly crafting and then deleting half a dozen causality collapsers or something.

**Comment by Mr3DAlien** [comment]
Yeah I tried it a few times now, and got one pearl but she was destroyed by the other rift... Maybe Azanor should remove the option for a rift to destroy this pearls and void seeds

**Comment by androshalforc** [comment]
sorry after watching your video i realized that the items i received were void seeds not pearls i have not gotten far enough to need them yet so i wasn't paying attention to what they were.

**Comment by lOmicronl** [comment]
Just don't overload your chunk with such absurd amount of flux that you instantly get new rifts. You'll notice that "if you are not careful with what you are doing, bad things will happen to you" is a theme of the mod... so ruining your loot by being carelessly excessive is par for the course =P

**Comment by Mr3DAlien** [comment]
The reason to overload the Chunk is to get a pearl. You need to do this to get this Item. I am not speaking about removing the option for rifts to destroy items. I am talking about removing the option of destroying the pearl. Because it is really rare and hard to get. And it really sucks when you get a pearl and then it gets destroyed.

**Comment by Morinor** [comment]
I got a pearl every 10th rift that I blew up

**Comment by SirMoogle** [comment]
It took me a while but I ended up getting my primordial pearl after blowing up countless rifts. Just have a steady source of causality collapsers and pray to RNGesus.

**Comment by monkeybutt32** [comment]
Hi, i am playing sky odyssey. I have stacks and stacks of seeds. I believe the original poster when stating 1000 causality pearls tossed. I now have made an afk rift farm. left running over night. Still no pearl. Any ideas?

![2019-12-10_19 03 38](https://user-images.githubusercontent.com/23014509/70582333-cb126b80-1b7f-11ea-85bf-06c69a91819d.png)

**Comment by monkeybutt32** [comment]
okay, the crafting recipe does not cycle the other smaller variants, mote and module. I have been waiting for the full pearl the whole time to match the recipe. no one in youtube or wiki specifically states, "they are the same thing and can be used in place of a full pearl. " I got pissed after days of trying and just threw one and the infusion altar. LOL it worked.
